do.sh

```diff
@@ -53,19 +81,37 @@ function generate() {
 
 function install_deps() {
     echo "-- Installing dependencies on all nodes"
-    ansible-playbook ${ovn_fmn_playbooks}/install-dependencies.yml -i ${hosts_file}
+    local _install_dep_playbook
+    if is_rpm_based
+    then
+        _install_dep_playbook=install-dependencies-rpm.yml
+    elif is_deb_based
+    then
+        _install_dep_playbook=install-dependencies-deb.yml
+    else
+        die_distro
+    fi
+    ansible-playbook ${ovn_fmn_playbooks}/${_install_dep_playbook} -i ${hosts_file}
 
     echo "-- Installing local dependencies"
-    if yum install docker-ce --nobest -y
+    if is_rpm_based
     then
-        systemctl start docker
+        if yum install docker-ce --nobest -y
```

Comment:
I know it was like this before your change but we (RedHat) don't use docker-ce anymore downstream.  We use podman downstream (and in CirrusCI).  I wonder if we should just remove docker-ce all together and simplify things..

Reply:
Sure, I'll add a separate commit for that

Reply:
Fixed in https://github.com/dceara/ovn-heater/pull/157/commits/d92932f6b4eae36dbd3a8f4dff61c0c709086b4b